app/controllers/v0/institutions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module V0
+  class InstitutionsController < ApplicationController
+    skip_before_action :authenticate
+
+    # def index
+    #   render json: [Institution.all.limit(3).sample]
+    # end
+
+    # e.g. /v0/gibct/institutions/autocomplete?term=harv
+    #
+    def autocomplete
+      search_term = params[:term]
+      render json: Institution.autocomplete(search_term)
+    end
+
+    # e.g. /v0/gibct/institutions/profile?facility_code=31800121
+    #
+    def profile
```

Comment:
I will chime in and agree... in general my preference is to always stick to the standard controller actions and only introduce custom actions if it truly makes sense. This seems like a great candidate for a nested resource -- perhaps using `resource` vs `resources` if there is only 1 profile and not multiple.